ipa-core/src/protocol/ipa_prf/malicious_security/hashing.rs

```diff
@@ -53,10 +53,16 @@ where
 /// This function takes two hash a vector of field elements into a single field element
 /// # Panics
 /// does not panic
-pub fn hash_to_field<F>(left: &Hash, right: &Hash) -> F
+pub fn hash_to_field<F>(left: &Hash, right: &Hash, exclude_to: u128) -> F
 where
-    F: Field + FromRandomU128,
+    F: PrimeField + FromRandomU128,
 {
+    assert!(F::BITS <= 64, "Field size is not sufficiently small");
```

Comment:
"Field size too large compared to the 128 bits of entropy, which will result in excessive bias when converting to a field element"